README.md

```diff
@@ -34,7 +45,7 @@ Please cite our work:
 }
 ```
 
-## How to run
```

Comment:
Keep `##` and use `###` and `####` if you want to create more subtopics
(the more you add # characters (Up to 5 # characters) the more the title gets smaller.

Reply:
i think that "how to run" isnt a subtopic, but the central capter in the whole readme, thats why just #.